sunpy/io/tests/test_filetools.py

```diff
@@ -18,13 +18,17 @@
 pytestmark = pytest.mark.filterwarnings("ignore:Invalid 'BLANK' keyword in header")
 
 
+@pytest.mark.remote_data
```

Comment:
Does this need this now?

Reply:
But we didn't need it before, so is that due to fsspec?

Reply:
Maybe. I was wondering how it was being accessed without the internet access decorator. It threw an internet error decorator error when testing, and it seemed logical to put it, and didn't break anything else, hence I put it.

Reply:
I see now why the decorator was not put, it was already mocked to prevent network access. I will try to run it again using the earlier test.